system-addon/content-src/components/Sections/Sections.jsx

```diff
@@ -18,6 +18,14 @@ function getFormattedMessage(message) {
 }
 
 export class Section extends React.PureComponent {
+  get numRows() {
+    const {rowsPref, maxRows, Prefs} = this.props;
+    if (rowsPref && Prefs.values[rowsPref] <= maxRows) {
```

Comment:
or should we let a user set the pref to 100 rows of highlights? 🤔

Reply:
to clarify ^. If the user went through the trouble of going into `about:config` to try to get more rows of Highlights. Should we respect that? We might run out of highlights rows from the feed but in most cases we should have enough for more than the max I think.

Reply:
I think about:config overrides is fine. The about:preferences UI will show a blank entry, and something too big will just have a bunch of placeholders where someone adjusting about:config can adjust again. I guess for Pocket, it'll happen to work for 7 rows as we get 20 stories? (Now someone will file a bug saying there's too many empty cards!)